airflow/task/task_runner/standard_task_runner.py

```diff
@@ -82,13 +86,21 @@ def _start_by_fork(self):  # pylint: disable=inconsistent-return-statements
             setproctitle(proc_title.format(args))
 
             try:
+                os.close(rpipe)
+                sys.stdout.flush()
+                # replace fds for the stdout/stderr with the newly created pipe
+                os.dup2(wpipe, 1)
+                os.dup2(wpipe, 2)
```

Comment:
It’s probably better to use `sys.__stdout__.fileno()` and `sys.__stderr__.fileno()`. They are basically always `1` and `2` on POSIX but are at least more readable.